packages/larva-tokens/style-guides/sportico.json

```diff
@@ -0,0 +1,77 @@
+{
+	"imports": [
```

Comment:
This is also a stray JSON - delete. There is something weird happening in the Gulpfile, potentially